pages/common/gitleaks.md

```diff
@@ -0,0 +1,32 @@
+# gitleaks
+
+> A scanner for detecting secrets and API keys leaked in Git repositories.
+> For more information, see: <https://github.com/gitleaks/gitleaks>.
+
+- Scan a remote repository:
+
+`gitleaks detect --repo-url {{https://github.com/username/repository.git}}`
+
+- Scan a local directory:
+
+`gitleaks detect --source {{path/to/repo}}`
```

Comment:
```suggestion
`gitleaks detect --source {{path/to/repository}}`
```
We generally prefer verbose descriptions and placeholders wherever possible.

Reply:
I think we should make this clearer in the style guide, it currently says:

> #### Naming
> - Use short but descriptive placeholders, such as {{path/to/source_file}} or {{path/to/wallet.txt}}.

We could clarify this improving the example:

> #### Naming
> - Use short but descriptive placeholders, such as {{path/to/source_file}} or {{path/to/wallet.txt}} instead of {{path/to/src}} or {{path/to/file.txt}}.